beta/src/views/discover/index.js

```diff
@@ -17,6 +17,23 @@ function renderDiscover (state, emit) {
         The co-operative music streaming platform.<br>
         Owned and run by members.
       </h2>
+      <div class="mv4">
+        <h2 class="di mr3 lh-title f3 fw1 v-mid">
+          The co-operative music streaming platform.<br />
+          Owned and run by members.
+        </h2>
+
+        ${!state.user.uid
+          ? html`
+              <a
+                class="link pv2 ph3 ttu ba b--mid-gray b--dark-gray--dark f6 b"
+                href="https://resonate.coop/join"
+              >
+                Join now
+              </a>
+            `
+          : null}
+      </div>
```

Comment:
@webwelten, thank you so much for your hard work on this. This will really help to provide an immediate route to signups for users.

I wanted to provide a slightly different option from a design perspective to just try to keep consistent with the previous text on the line. This has its pros and its cons, so I will defer to everyone else here as to what they prefer. 

That aside, the changes as they are now duplicate the text (L17-18 and L22-23) that precedes the Join Now button. This suggestion below also removes the extra text:

```suggestion
        <div class="mv4">
          The co-operative music streaming platform.<br />
          Owned and run by members.

          ${!state.user.uid
            ? html`
                <a href="https://resonate.coop/join">
                  Join now
                </a>.
              </h2>
              `
            : null}
        </div>
      </h2>
```

<img width="1792" alt="Screen Shot 2022-03-05 at 6 43 47 PM" src="https://user-images.githubusercontent.com/60944077/156904651-679cd8a0-49a3-473c-9f1f-20a32706e5d0.png">

Reply:
Hey Peter,
thanks for the comment and the suggestion you made. I also removed the redundant code.
I'm actually ok with both, yours seems to fit in well with the text. Who will decide that?
@hakanto ?